website/source/docs/platform/k8s/helm.html.md

```diff
@@ -56,7 +56,7 @@ $ cd consul-helm
 $ git checkout v0.1.0
 
 # Run Helm
-$ helm install --dry-run ./
+$ helm install --name consul .
```

Comment:
without this, the consul release gets random names like `sanguine-elk-consul`. Recommending everyone use the `consul` name will make it easier to write clear docs and will give a more professional impression to users